src/Form/Attribute/HasColorspace.php

```
@@ -0,0 +1,48 @@
+     * @param string|UnitEnum|null $value Color space (`limited-srgb` or `display-p3`), or `null` to remove the
+     * attribute.
+     *
+     * @throws InvalidArgumentException if the provided value is not valid.
+     *
+     * @return static New instance with the updated `colorspace` attribute.
+     *
+     * {@see Colorspace} for predefined enum values.
+     */
+    public function colorspace(string|UnitEnum|null $value): static
+    {
+        Validator::oneOf($value, Colorspace::cases(), 'colorspace');
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\UIAwesome\Html\Helper\Validator' in method 'colorspace'. (undefined)

(StaticAccess)